federation/vstoken.actions.githubusercontent.com/config.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 The Sigstore Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+url: https://vstoken.actions.githubusercontent.com
+contact: lorenc.d@gmail.com
```

Comment:
How does the contact work? Is this like the 'application developer'? I was just thinking it might be a good reason for us to get something like 'support@sigstore.dev' in place.

Reply:
whoops, it's not really used now. I'll put that.